### DISCUSSION TITLE
Is the currently installed (official) firmware important?

### DISCUSSION THREAD

**Discussion by Igor01-Tech** [alienatedsec/yi-hack-v5#375] [Q&A] [answered]
I just bought a two pack of the Yi Home 1080p cams. And they come with the Firmware 8.2.0.0A_2019122709941 out of the box. And I get a firmware update to Version 12.2.0.2_202212191129.

I just need the RTSP stream - nothing else. So do I need to upgrade the official firmware before flashing the Yi Hack V5? Or will it Work with the 8.2xxx version too?

**Comment by alienatedsec** [comment, marked as answer]
Don't upgrade the firmware as it may not be compatible with the hack. V5 is not for your camera

Check @roleoroleo repos for Allwinner

**Reply by Igor01-Tech** [reply]
Oh it isn't? But it's that "Yi Home 1080p" cam? What made you realize mine is not suitable?

**Reply by alienatedsec** [reply]
I know the base firmware numbers for the supported chipsets and yours is not there.

**Reply by Igor01-Tech** [reply]
Thanks! Maybe you should mention this in the readme? :)

**Reply by alienatedsec** [reply]
This repo and `yi-hack-v5` inherited supported version identification based on the first 4 characters of the serial number. To be honest, this is already described in the repo's readme. However, the YI firmware version numbers could also be referenced. @Igor01-Tech

**Reply by alienatedsec** [reply]
here you go @Igor01-Tech https://github.com/alienatedsec/yi-hack-v5/commit/d1997ae29ddeaf7d39d4d42c6da85901b7e56bcd

**Reply by Igor01-Tech** [reply]
This should be very helpful to many! Thanks!!

**Reply by jaracoll** [reply]
Hi all,

First of all, thanks for your huge effort @alienatedsec. It's amazing what you have achieved here. Also, it's important to highlight that your efforts help to reuse existing device and fight the planned obsolescence (which normally comes with app changes). In the end, we reduce waste and save money. Thanks for that!

I've acquired some second hand cameras (1080p AI+) which firmware is "...2022...". The firmware is not supported by the project. I would like to know if someone has **downgraded the camera** to a firmware compatible with the yi-hack-v5. If so, **how you did it?** I found some files in different repositories and webs. No one worked.

Many thanks!

**Reply by alienatedsec** [reply]
@jaracoll this model is likely a newer generation on a different chipset. Search for Allwinner hacks from @roleoroleo

**Reply by jaracoll** [reply]
Ok. Many thanks! Have a nice day
